docs/howto/kubernetes/gardener/kubectl.md

```diff
@@ -3,40 +3,64 @@ description: How to fetch, verify, and use your kubeconfig with kubectl.
-> clusters.
+To access your shoot cluster, choose between a *static* and a *certificate-based* kubeconfig file.
+
+* A **static** kubeconfig contains a [static token](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#static-token-file), provides `cluster-admin` privileges to the corresponding shoot cluster, and never expires.
+In {{brand}}, up until and including Kubernetes 1.26, you can generate a static kubeconfig for your shoot cluster.
+
+* A **certificate-based** kubeconfig is valid for a predefined amount of time, during which it can be used for accessing the corresponding shoot cluster with `cluster-admin` privileges.
+The credentials associated with this type of kubeconfig are client certificates with time-limited validity.
+Beginning with Kubernetes version 1.27, you will be able to issue certificate-based kubeconfig files only.
+
+To get a kubeconfig file for a shoot cluster, in the {{gui}} click on the cluster row to expand its properties and go to the *KubeConfig* tab.
+
+=== "Static kubeconfig"
+    ![KubeConfig tab in {{k8s_management_service}} Shoot view](assets/shoot_kubeconfig.png)
+=== "Certificate-based kubeconfig"
+    Notice that you may set the duration of the kubeconfig file validity.
+
+    ![KubeConfig tab in {{k8s_management_service}} Shoot view](assets/shoot_kubeconfig_cert_01.png)
+
+    The default validity period is one hour, but you may use the drop-down menu to select a different duration.
+
+    ![Select a validity period for kubeconfig](assets/shoot_kubeconfig_cert_02.png)
+
+    When satisfied with the duration, click the green button labeled *Generate config*.
+
+    ![Generate a certificate-based kubeconfig](assets/shoot_kubeconfig_cert_03.png)
+
+    Right below, you will see the contents of your dynamically generated kubeconfig.
+
+    ![Download the newly generated kubeconfig](assets/shoot_kubeconfig_cert_04.png)
+
+To download your kubeconfig, click the blue button labeled *Download KubeConfig*.
+In the default download folder of your local user account, you will get a configuration file named like so:
+
+```plain
+kubeconfig--<cluster_name>--<region_name>--<project_id>.yaml
+```
+
+Create a directory named `.kube` in your local user's home, then move the YAML file you downloaded into it.
+Rename the YAML to `config`, ending up with `~/.kube/config`.
+By default, `kubectl` searches for its configuration in `~/.kube/config`.
+If necessary, you can [modify this behavior](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig) by setting the `KUBECONFIG` environment variable.
+For example, if you want to retain the original filename exactly as downloaded, you will type something like the following:
+
+```shell
+export KUBECONFIG=~/.kube/kubeconfig--<cluster_name>--<region_name>--<project_id>.yaml
+```
+
+If you manage multiple Kubernetes clusters, you will probably prefer this approach.
 
 ## Verifying your kubeconfig
```

Comment:
This section needs an update, as well. It presently shows only a static kubeconfig.